nfv_ovs_dpdk.md

```diff
@@ -0,0 +1,262 @@
+# NFV OVS DPDK
+
+This document describes how to deploy EDPM nodes with OVS DPDK.
+
+## Procedure Overview
+
+1. Configure kernel args and tuned parameters
+2. Configure OVS DPDK parameters
+3. Configure the networks with DPDK interfaces of the EDPM nodes
+2. Configure Nova and Neutron parameters.
+
+In order to complete the above procedure, the `services` list of the
+`OpenStackDataPlaneNodeSet` CR needs to be edited.
+
+## OpenStackDataPlaneNodeSet services list
+
+EDPM nodes can be configured by creating an
+`OpenStackDataPlaneNodeSet` CR which the
+[dataplane-operator](https://openstack-k8s-operators.github.io/dataplane-operator)
+will reconcile when an `OpenStackDataPlaneDeployment` CR is created.
```

Comment:
I would say `will deploy' instead of `will reconcile`, since technically the `OpenStackDataPlaneNodeSet` is reconciled even without an `OpenStackDataPlaneDeployment` (it just won't process certain things yet).  Or maybe we can say something like `will reconcile to create OpenStackDataPlaneService resources`, because I _think_ the services are created when the `OpenStackDataPlaneDeployment` is created.

Reply:
ack